src/components/shared/search/Search.module.scss

```diff
@@ -0,0 +1,46 @@
+@use '@variables' as *;
+@use '@mixins' as mixins;
+
+.container {
+  position: absolute;
+  top: 100%;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.form {
+  display: flex;
+  border-bottom: 1px solid black;
+  width: 100%;
+  max-width: 400px;
+  padding: 0.25rem 0;
+
+  input {
+    border: none;
+    width: 100%;
+    font-size: 1rem;
+    @include mixins.openSans;
+  }
+
+  input:focus {
+    outline: none;
+  }
+}
+
+@media (max-width: $tablet) {
+  .form {
+    input {
+      font-size: 0.875rem;
```

Comment:
If were going to use rems, can you create mixins for each rem so we know what pixel value it corresponds to?
I don't know what 0.875 rems means, so if you create a mixin called @mixin 16px, ill know what size this font is intended to be